src/tapeimage.cpp

```diff
@@ -255,8 +261,7 @@ record_index::find(std::int64_t n, iterator hint) const noexcept (false) {
         return hint;
     }
 
-    /* don't consider the first ghost node as a candidate */
-    const auto begin = std::next(this->begin());
+    const auto begin = this->begin();
     const auto end   = this->end();
```

Comment:
There actually seem to be not much need now for introduction of these variables

Reply:
True, but it slightly cleans up the actual search, so I don't think it's so bad.